timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/v5/jsonldimport/GenerateRdfPatchFromJsonLdEntity.java

```diff
@@ -0,0 +1,132 @@
+package nl.knaw.huygens.timbuctoo.v5.jsonldimport;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import nl.knaw.huygens.timbuctoo.v5.bdbdatafetchers.dto.CursorQuad;
+import nl.knaw.huygens.timbuctoo.v5.dataset.Direction;
+import nl.knaw.huygens.timbuctoo.v5.dataset.PatchRdfCreator;
+import nl.knaw.huygens.timbuctoo.v5.dataset.QuadStore;
+import nl.knaw.huygens.timbuctoo.v5.filestorage.exceptions.LogStorageFailedException;
+import nl.knaw.huygens.timbuctoo.v5.rdfio.RdfPatchSerializer;
+import nl.knaw.huygens.timbuctoo.v5.util.RdfConstants;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+public class GenerateRdfPatchFromJsonLdEntity implements PatchRdfCreator {
+  private Entity[] entities;
+  private List<CursorQuad> toReplace = new ArrayList<>();
+
+
+  @JsonCreator
+  public GenerateRdfPatchFromJsonLdEntity(@JsonProperty("entities") Entity[] entities) {
```

Comment:
I think this method also requires a `@JsonProperty("toReplace") List<CursorQuad> toReplace` to function correctly (you can then also make the fields final)